src/pages/about/overview.md

```diff
@@ -41,8 +41,8 @@ IBM Watson Annotator for Clinical Data is a REST API service that detects medica
 When you send unstructured data to the service to be analyzed and designate the desired annotators to employ,
 the service will route your unstructured data through the designated annotators and return the medical concepts detected within your unstructured data.
 
-1. Designate which [available annotators](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-overview#available-annotators) to employ in analyzing your unstructured data.
-   This designation is defined as an annotator flow. See [Analyzing Text](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-analyze_text) for more details.
+1. Designate which [available annotators](/clouddocs/overview/) to employ in analyzing your unstructured data.
```

Comment:
This one for "available annotators" should go to this link to make it a bit more specific: https://ibm.github.io/acd-containers/clouddocs/overview/#available-annotators. I think you should be able to use the internal version of that, such as /clouddocs/overview/#available-annotators.